html/browser/common.css

```diff
@@ -173,35 +271,47 @@ h4
 	font-family: Verdana, Geneva, sans-serif;
 }
 
-.good
-{
-	color: #00ff00;
-}
-
-.mild
-{
-	color: #e0d000;
-}
-
-.average
-{
-	color: #f09000;
-}
-
-.bad
-{
-	color: #ff0000;
-}
-
-.highlight
-{
-	color: #8BA5C4;
-}
-
-.dark
-{
-	color: #272727;
-}
+.black{color:#0d0d0d}
+.white{color:#ffffff}
+.red{color:#dd3333}
+.orange {color:#f37827}
+.yellow {color:#fbd814}
+.olive{color:#c0d919}
+.green{color:#22be47}
+.teal {color:#00c5bd}
+.blue {color:#238cdc}
+.violet {color:#6c3fcc}
+.purple {color:#a93bcd}
+.pink {color:#e2439c}
+.brown{color:#af6d43}
+.grey {color:#7d7d7d}
+.gray {color:#7d7d7d}
+.good {color:#62b62a}
+.average{color:#f1951d}
+.bad{color:#dd3333}
+.label{color:#8496ab}
+.highlight{color:#8496ab}
+.dark {color:#272727}
+
+.bgblack {background-color:#000000!important}
+.bgwhite {background-color:#d9d9d9!important}
+.bgred {background-color:#bd2020!important}
+.bgorange{background-color:#d95e0c!important}
+.bgyellow{background-color:#d9b804!important}
+.bgolive {background-color:#9aad14!important}
+.bggreen {background-color:#1b9638!important}
+.bgteal{background-color:#009a93!important}
+.bgblue{background-color:#1c71b1!important}
+.bgviolet{background-color:#552dab!important}
+.bgpurple{background-color:#8b2baa!important}
+.bgpink{background-color:#cf2082!important}
+.bgbrown {background-color:#8c5836!important}
+.bggrey{background-color:#646464!important}
+.bggray{background-color:#646464!important}
+.bggood{background-color:#4d9121!important}
+.bgaverage {background-color:#cd7a0d!important}
+.bgbad {background-color:#bd2020!important}
+.bglabel {background-color:#657a94!important}
```

Comment:
Please move any CSS changes not critical for the PR into a separate pull request.